pipeline/trusted-resources/cmd/sign/main.go

```diff
@@ -32,18 +32,22 @@ import (
 	"github.com/sigstore/sigstore/pkg/signature/kms"
 	"github.com/tektoncd/experimental/pipelines/trusted-resources/pkg/trustedtask"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/webhook/resourcesemantics"
 	"sigs.k8s.io/yaml"
 )
 
 var (
-	cosignKey  = flag.String("ck", "", "cosign private key path")
-	kmsKey     = flag.String("kms", "", "kms key path")
-	taskFile   = flag.String("ts", "", "YAML file path for tekton task")
-	targetDir  = flag.String("td", "", "Dir to save the signed files")
-	targetFile = flag.String("tf", "signed.yaml", "Filename of the signed file")
+	cosignKey    = flag.String("ck", "", "cosign private key path")
+	kmsKey       = flag.String("kms", "", "kms key path")
+	resourceFile = flag.String("rf", "", "YAML file path for tekton resources")
+	kind         = flag.String("kd", "Task", "The kind of the signed object")
```

Comment:
```suggestion
	kind         = flag.String("kd", "Task", "The kind of the signed object. Supported values: [Task, Pipeline]")
```

(we may also want to make this case insensitive in another PR)

Reply:
OK! I will add a todo here